src/Language/Lexer.php

```diff
@@ -522,8 +525,27 @@ private function readString($line, $col, Token $prev)
                                 'Invalid character escape sequence: \\u' . $hex
                             );
                         }
+
                         $code = hexdec($hex);
+
+                        // UTF-16 surrogate pair detection and handling.
+                        $high_order_byte = $code >> 8;
```

Comment:
Hm, we are supposed to have camelCase for variable names via **phpcs**. @simPod any ideas why it would miss this snake case?

Reply:
@vladar Yea, sorry... Will be in v7, introduced here https://github.com/doctrine/coding-standard/pull/128 (by me what an accident! 😂 )

V7 of Doctrine CS is going to be cool, many new things to check for. I'll send PR here when it's released.

Reply:
I manually updated the variables to lowerCamelCase in the meantime